build.gradle

```diff
@@ -34,6 +35,11 @@ repositories {
     }
 }
 
+configurations {
+    // prevents libs from including commons-logging, which Android already does
+    all*.exclude group: 'commons-logging', module: 'commons-logging'
```

Comment:
addresses the hundreds of instances of the following warning that pop up at build time

> WARNING: Dependency commons-logging:commons-logging:1.1.1 is ignored for qabetaDebug as it may be conflicting with the internal version provided by Android.